app/controllers/book_loans_controller.rb

```diff
@@ -5,14 +5,14 @@ class BookLoansController < ApplicationController
   def create
     respond_to do |format|
       if @book_loan.save
-        Publishers::LoanBookPublisher.new(@book_loan.attributes).publish
+        Publishers::LoanBookPublisher.new(@book_loan.attributes, 'book_loans').publish
```

Comment:
Te wywołania Trzeba by wydzielić do osobnych metod i najlepiej serwisu. Opisałem to w innym PR :)